src/hooks/useBasisForm/useBasisForm.ts

```diff
@@ -15,15 +15,17 @@ import { setBasisError } from "./setBasisError";
 
 type UseBasisOptions<TFieldValues extends FieldValues = FieldValues> = {
   defaultValues?: UnpackNestedValue<DeepPartial<TFieldValues>>;
+  shouldUnregister?: boolean;
 };
 
 export function useBasisForm<FieldValues>(
   options: UseBasisOptions<FieldValues> = {}
 ) {
-  const { defaultValues } = options;
+  const { defaultValues, shouldUnregister } = options;
   const methods = useForm<FieldValues>({
     mode: "onBlur",
     defaultValues,
+    shouldUnregister,
```

Comment:
Wondering if we should just pass all the options?
```
const methods = useForm<FieldValues>({
  mode: "onBlur",
  ...options
});
```
@zecuria Any thoughts?

Reply:
I would say no, the point of this is to provide a unified experience. If there are use cases for each of them then we can easily add them.